frontend/restInterfaces/SequenceRestInterface.ts

```diff
@@ -0,0 +1,129 @@
+import { BaseRestInterface } from './BaseRestInterface';
+import { ISlide } from '../../model/slide/ISlide';
+import { ISequence } from '../../model/sequence/ISequence';
+import { ISequenceWithSlides } from '../../model/sequence/ISequenceWithSlides';
+
+/**
+ * Implements communication with the Server concerning all Sequence-requests
+ */
+export class SequenceRestInterface extends BaseRestInterface {
+  /**
+   * Sends a request to backend to add a new Sequence
+   * @param title the title of the sequence the user is trying to add
+   * @returns code of the sequence
+   */
+  public static async addSequence(title: string): Promise<string> {
+    return await BaseRestInterface.put<string>('/sequence/', title);
```

Comment:
Same as above, `{ code: string }` instead of `string` inside the generic, and `{ name: title }` as parameter